README.md

```diff
@@ -798,7 +798,7 @@ In these cases, conditional attributes works well. We can pass `if` option to `a
 
 ```ruby
 class User
-  attr_accessor :id, :name, :email, :created_at, :updated_at
```

Comment:
(This is mostly unrelated, but I'm removing the `:created_at` and `:updated_at` `attr_accessor`s here because they aren't really relevant to the example and I think are therefore unhelpfully distracting rather than helpful.)